jax/experimental/jet.py

```diff
@@ -239,6 +239,22 @@ def _expm1_taylor(primals_in, series_in):
   return lax.expm1(x), series_out
 jet_rules[lax.expm1_p] = _expm1_taylor
 
+def _pow_taylor(primals_in, series_in):
+  u_, r_ = primals_in
+  u_terms, r_terms = series_in
+  u = [u_] + u_terms
+  r = [r_] + r_terms
+
+  logu_, logu_terms = _log_taylor((u_, ), (u_terms, ))
+
+  v_, v_terms = _bilinear_taylor_rule(lax.mul_p, (logu_, r_), (logu_terms, r_terms))
+
+  v_, v_terms = _exp_taylor((v_, ), (v_terms, ))
+
+  return v_, v_terms
```

Comment:
If you're going to define things this way, I'd just call back into `jet`:
```suggestion
  return jet(lambda x, y: lax.exp(y * lax.log(x), primals_in, series_in)
```

Reply:
Haha, I feel silly now.  We started by trying to derive a giant monolithic formula for pow(x, y), but we already had all the ingredients!

Reply:
@shoyer Thanks for pointing this out! I realized that I should change the `primal_out` propagated through the `exp` rule, so I incorporated your suggestion for propagation through `mul` and `log` in [77b7d9b](https://github.com/google/jax/pull/2652/commits/77b7d9be1924d4c73ab8884d9d9306245b6fbe21) (added you as a co-author)

Reply:
@shoyer yeah! We're still unclear if there is a benefit from writing the expanded rules for primitives that can be composed of other primitives vs just calling jet on that composition like you have here.